fairscale/optim/oss.py

```diff
@@ -565,24 +564,6 @@ def _broadcast_params(self) -> None:
         if last_work_handle:
             last_work_handle.wait()
 
-    def _consume_work_handles(self) -> None:
```

Comment:
since the tensor views change (broadcast buckets use tensor views) this was not being used in OSS anymore, just in ShardedDDP, I figured it was cleaner to move it there